easybuild/easyconfigs/m/msgpack-c/msgpack-c-3.3.0-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,30 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2021/04
+easyblock = 'CMakeMake'
+
+name = 'msgpack-c'
+version = '3.3.0'
+
+homepage = 'http://msgpack.org/'
+description = """MessagePack is an efficient binary serialization format, which lets you exchange
+data among multiple languages like JSON, except that it's faster and smaller.   
+Small integers are encoded into a single byte while typical short strings       
+require only one extra byte in addition to the strings themselves."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = [GITHUB_SOURCE]
+github_account = 'msgpack'
+sources = ['cpp-%(version)s.tar.gz']
+checksums = ['754c3ace499a63e45b77ef4bcab4ee602c2c414f58403bce826b76ffc2f77d0b']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+    ('binutils', '2.35'),
+]
+
+sanity_check_paths = {
+    'files': [('lib/libmsgpackc.%s' % SHLIB_EXT)],
```

Comment:
There doesn't happen to be a cpp version of the library as well that we should check for?